docs/results.md

```diff
@@ -45,6 +45,8 @@ Version   | Approach    | Micro f1 | Description
 2020.10.13 | tfidf-svm | 0.72 | same as 2020.5.2 but train on title+synopsis and test on title+synopsis+lay summary+research question
 2020.10.14 | tfidf-svm | 0.72 | same as 2020.5.2 but train on title+synopsis and test on title+synopsis+lay summary
 2020.10.15 | tfidf-svm | 0.71 | same as 2020.5.2 but train on title+synopsis and test on title+synopsis+research_question
+2020.10.16 | tfidf+onehot_team | 0.71 | similar to 2020.5.2 but we add team as one hot variable to the features. as data get shuffled differently it seems as the performance stays the same but it actually increases 1% compared to removing the team variable
+2020.10.17 | tfidf+onehot_scheme | 0.68 | similar to 2020.10.16 but this time we add scheme as one hot variable.
```

Comment:
what do you think is going on? the scheme has confused the model, needs to train for longer?

Reply:
just a thought - but do you think maybe the scheme field needs cleaning, I wonder whether there is a lot of missing data or inconsistent scheme names? I see you do `processed_data.replace("No Data Entered", "").fillna("")` so maybe that would solve things.

Reply:
you are wondering why performance drops instead of saying the same? this is what i was discussing in the slack channel saying that i find it unituitive when this happens even though it is a standard effect. scheme in our case does not carry much more information and possibly it only adds noise and dimensions to the dataset.

how do you suggest cleaning better? i am keen to further explore this if you have any ideas. atm scheme is treated as empty, of course we can deploy other imputation strategies but in terms of cleaning the data that exists, i think we are fine

Reply:
oh yeh, I remember that conversation now. Yeh would assume it would stay the same so 3% down seems quite a bit!

I don't have any cleaning suggestions, just a thought.